tests/test_library_vmaf_eval.py

```
@@ -0,0 +1,374 @@
+
+    result = subprocess.run(
+        ["bash", str(BASH_SCRIPT), str(config)],
+        capture_output=True,
+        text=True,
+        env=env,
+        timeout=30,
+    )
+    assert result.returncode == 0, (
+        f"script failed:\nstdout={result.stdout}\nstderr={result.stderr}"
+    )
+    lines = [l for l in result.stdout.strip().splitlines() if l]
```

Ruff (0.16.2)

[error] Ambiguous variable name: `l`

(E741)